Assets/Scripts/Puzzle/CoOpTriggerHandler.cs

```diff
@@ -21,48 +21,29 @@ public class CoOpTriggerHandler : MonoBehaviour
 
         public void GhostInteract()
         {
-            if (!canActivate)
-            {
-                return;
-            }
-            
+            if (!canActivate) return;
+            if (!ghostIsInteracting) return;
```

Comment:
You can combine these into one if statement with || and also I think it's always more readable to use bracers on all if statements.